### DISCUSSION TITLE
Speed up NRI/NTI calculations by paralellization in large datasets

### DISCUSSION THREAD

**Discussion by eperezv** [ChiLiubio/microeco#400] [Ideas]
Hi,

I've tried to perform NRI/NTI calculations using microeco and I noticed it can work very slow when working with large datasets. I see there is a parallel implementation of the index calculation in [MicEco ](https://github.com/Russel88/MicEco/) package. I think it could be great if microeco could implement parallel computing of these indexes. 
Thank you

**Comment by ChiLiubio** [comment]
Thank you very much. I will study how to optimize it.
